### PR TITLE
Add aadGroupId to TeamInfo

### LINKED ISSUE #4069: TeamInfo type does not expose aadGroupId (dotnet)
See [parent](https://github.com/microsoft/botframework-sdk/issues/5870)

### PR DESCRIPTION
**Pull request by EricDahlvang**
Fixes #4069

### CODE REVIEW

**Comment by EricDahlvang** [comment]
@gabog TeamInfo is a field on TeamsChannelData

With this PR, the following will return the aadObjectId for the Team:

`turnContext.Activity.GetChannelData<TeamsChannelData>().Team.AadGroupId`

File: libraries/Microsoft.Bot.Schema/Teams/Generated/TeamInfo.cs [hunk 1]
```diff
@@ -55,5 +55,10 @@ public TeamInfo(string id = default(string), string name = default(string))
         [JsonProperty(PropertyName = "name")]
         public string Name { get; set; }
 
+        /// <summary>
+        /// Gets or sets aadGroupId of team.
```
Comment (gabog) [inline libraries/Microsoft.Bot.Schema/Teams/Generated/TeamInfo.cs:59]: Can we reword this as Gets or sets the Azure AD group ID?

Comment (EricDahlvang) [reply libraries/Microsoft.Bot.Schema/Teams/Generated/TeamInfo.cs]: changed the comment
